docs/blog/posts/2023-05-16-EDGARmirror.md

```diff
@@ -161,16 +167,16 @@ Syntax supports the following:
+WRDS provides [tables](https://wrds-www.wharton.upenn.edu/pages/get-data/wrds-sec-analytics-suite/wrds-sec-linking-tables/){:target="_blank"} you can use to join datasets with different IDs across time. It is a good idea to [ask a GSB librarian](https://www.gsb.stanford.edu/library/research-support/ask-us/){:target="_blank"} for help on how exactly to use these tables so that you can get the right company matches especially if you are looking at a large span of time. Linking tables include:
 
 - Historical CIK-CUSIP Link Table
 - Historical Company Names of SEC Filers
 - Historical GVKEY-CIK Link Table
 
 ### Bag of Words
 
-WRDS <a href="https://wrds-www.wharton.upenn.edu/pages/get-data/wrds-sec-analytics-suite/wrds-sec-text-analysis/wrds-bag-words/" target="_blank">Bag of Word (BoW) interface</a> provides WRDS SEC Analytics Suite subscribers an alternative way to perform content analysis against company SEC filings. A frequency count of all words in SEC filings and amendments from 1993 through 2017 was prepared. Users can query word counts across all filings, or against specific company filings. Total word occurrence per filing, cosine similarity, Jaccard similarity, minimum edit distance between current and previous filings, word stemming and lemmatization, English word, positive or negative word, stop word, geographic, company name, patent related (requires KtMINE subscription) word indicators etc. are also available. 
-You can read more about how words are contructed and what summary measures were applied in the <a href="https://wrds-www.wharton.upenn.edu/documents/1081/WRDS_Bag_of_Words_Manual.pdf" target="_blank">manual</a>. 
+WRDS [Bag of Words (BoW) interface](https://wrds-www.wharton.upenn.edu/pages/get-data/wrds-sec-analytics-suite/wrds-sec-text-analysis/wrds-bag-words/){:target="_blank"} provides WRDS SEC Analytics Suite subscribers an alternative way to perform content analysis against company SEC filings. A frequency count of all words in SEC filings and amendments from 1993 through 2022 was prepared. Users can query word counts across all filings, or against specific company filings. Total word occurrence per filing, cosine similarity, Jaccard similarity, minimum edit distance between current and previous filings, word stemming and lemmatization, English word, positive or negative word, stop word, geographic, company name, patent related (requires KtMINE subscription) word indicators etc. are also available. 
+You can read more about how words are contructed and what summary measures were applied in the [manual](https://wrds-www.wharton.upenn.edu/documents/1081/WRDS_Bag_of_Words_Manual.pdf){:target="_blank"}. 
 
 Here are some of the modules available:
 ```
```

Comment:
I would maybe add a Summary to say something about the whole page. It ends abruptly.